otelib/backends/python/__init__.py

```diff
@@ -2,13 +2,13 @@
 
 from .dataresource import DataResource
 from .filter import Filter
-from .function import Function
```

Comment:
Revert this. We still want to include `Function`.

Also, add in the "Parser" strategy import here - otherwise it makes no sense to include it in the `__all__` variable below.